src/lib/update-cors-cloud.spec.ts

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import { updateCorsConfig } from './update-cors-cloud';
+
+const mock = new MockAdapter(axios as any);
+
+describe('libUpdateCorsCloud', () => {
+  beforeEach(() => {
+    mock.onPut('/global-config/services/CorsService').reply(201, { _id: '123' });
```

Comment:
the CORS url should also include the name of the configuration to be created/updated - e.g: 
`/global-config/services/CorsService/my-cors-config`
Also, if the specific configuration does not exist it will be created with a PUT request and the response will be 201 "Created". If the configuration already exists, it will be updated and the response status will be 200 "OK"...